en/V5-Hardware_Platform_Requirements.md

```diff
@@ -2,9 +2,9 @@
 
 ## Control Objective
 
-The secure hardware configuration is one of the fundamentals of the embedded device security. The device code can be written and configured in the most secure way, but it cannot help as long as hardware has backdoors in a form of open debug ports or other undocumented debug features.
+Hardware is more difficult and more costly to compromise and subvert than software. Therefore hardware security can provide a robust foundation for embedded device security. On the other hand, hardware that contains backdoors or undocumented debug features can completely compromise the security of the entire device. 
 
-The purpose of the controls listed in this section is to ensure that as long as hardware is available for secure configuration, it is been configured in the maximum possible secure way. This includes disabling or securing debug interfaces, setting up all existing alarms and sensors mechanism to combat tampering, use anti-cloning hardware protection, like OTP fuses, use MMU (Memory Management Unit) for secure process isolation.
+The purpose of the controls listed in this section is to ensure that as long as hardware is available for secure configuration, it is been configured in the most secure way possible. This includes disabling or securing debug interfaces, setting up all existing alarms and sensors mechanisms to combat tampering, using anti-cloning hardware protection such as OTP fuses, and the use of the MMU (Memory Management Unit) for secure process isolation.
```

Comment:
Minor edit as noted in a previous comment - Please use "in this chapter" versus "in this section"

Change "sensors" to singular "sensor mechanisms"